.teamcity/src/main/kotlin/common/performance-test-extensions.kt

```diff
@@ -74,7 +74,10 @@ fun BuildSteps.substDirOnWindows(os: Os) {
         script {
             name = "SETUP_VIRTUAL_DISK_FOR_PERF_TEST"
             executionMode = BuildStep.ExecutionMode.ALWAYS
-            scriptContent = """dir p: || subst p: "%teamcity.build.checkoutDir%" """
+            scriptContent = """
+                subst p: /d
```

Comment:
Does this succeed, even if the drive does not exist?

Reply:
Why don't we use something like `dir p: && subst p: /d` instead?

Reply:
The results are same.

Reply:
We already do `dir p: && subst p: /d instead?` when we remove the subst drive in after the build, so I would expect us to do this before the biuld as well. I think the difference is the color of the step (maybe?): 
![image](https://user-images.githubusercontent.com/423186/110593932-7b549280-817c-11eb-9f5a-e4b03963f774.png)

I think if there are any warnings, the step will have this orange color.